tests/src/Kernel/SmsFrameworkProviderTest.php

```diff
@@ -383,6 +388,7 @@ public function testEventsQueueIncoming() {
     $this->container->get('cron')->run();
 
     $expected[] = SmsEvents::MESSAGE_INCOMING_PRE_PROCESS;
+    $expected[] = 'incomingEvent'; // Plugin incoming event.
```

Comment:
It looks wierd that this is the only one without a constant defined in SmsEvents? Is it not a real event like the others?